### ISSUE TITLE
Meta: improve issue tracker

### ISSUE THREAD

**Issue by gterzian** [servo/servo#26882]
I think it might be a bit hard for someone not deeply involved in the project to understand what the status of an open issue is. 

I'm thinking people not looking for a "first issue", or for something with mentoring available, since those issues will be pretty clear, rather all the other issues that are opened, and sometimes comes with a bunch of comments but it's not clear what their status are. 

In particular, it might be hard to know whether it's worth trying to fix the issue based on what has been written in it, or whether it's still effectively being triaged. 

Perhaps we should add some labels, so someone wanting to work on an issue as a clearer idea of what to do(either further triage the issue, or actually start working on it based on the comments in the issue?)

**Comment by gterzian** [comment]
As an example: https://github.com/servo/servo/issues/26876

Based on my comments, I'd like to add some label like "triaged" or "has-potential-solution", something labelling the issue as something that has a proposed solution and that could be picked-up by someone wanting to work on it(but it's not necessarily as "easy" issue). 

Then again, let's say someone disagrees with my comments, the "in-triage" label could be re-added and the labels I would have added removed. 

Currently this is still possible to understand by reading the comments, but that requires some knowledge of the project I think, and it's also not filterable by people looking for a well-triaged issue to work on.

**Comment by CYBAI** [comment]
My personal perspective is, it might be good to have a `C-needs triage` label.

In my understanding, `A-` labels like `A-input`, `A-content/dom` and so on are for triage so contributors should be able to know what the issue for by the `A-`label.

However, when it's an issue which has its own upstream, we might be able to add a new label like `S-blocked-on-external` for it? (IIUC, we will need to fix #26876 to webrender?)

I think adding a `C-has potential solution` (like `C-has patch` but not has a patch yet) is good which should let everyone know there's a potential solution in comments.

(btw, we might want to update https://github.com/servo/servo/wiki/Tag-label-names-and-definitions wiki page to align our existing labels?)

**Comment by jdm** [comment]
I agree, it sounds like it could be useful to have a label that means "there's a description of a possible solution".

**Comment by gterzian** [comment]
> a label that means "there's a description of a possible solution".

How about `C-ready-for-assignment`?

**Comment by gterzian** [comment]
> How about C-ready-for-assignment?

There is already a `C-looking-for-assistance`, which could be used for that purpose. Also there is a `C-needs-investigation`, which could be used to mark as an issue that, well, requires more investigation/triage.
